ClientIdentifier.ts

```diff
@@ -2,7 +2,7 @@ import { Identifier } from "authly"
 
 export abstract class ClientIdentifier {
 	private constructor() {}
-	private static valueCache: string
+	private static valueCache: string | null
```

Comment:
Use `undefined` instead of `null`:
```
private static valueCache?: string
```

Reply:
That gives me this:

Type 'string | null' is not assignable to type 'string | undefined'.
  Type 'null' is not assignable to type 'string | undefined'.

Reply:
Ẁhere? Just append `|| undefined` to that expression.